R/tools.R

```diff
@@ -192,7 +192,7 @@ build_js <- function(watch = FALSE) {
   }
 }
 
-# nolint start
+# nolint start: line_length_linter
```

Comment:
this is interesting, didn't know you can use `nolint start` for a specific rule

Reply:
New feature in `lintr 3.0.0` :slightly_smiling_face: